backend/src/main/java/fk/prof/backend/model/association/impl/ZookeeperBasedBackendAssociationStore.java

```diff
@@ -396,13 +412,15 @@ private void loadDataFromZookeeperInBackendLookup() throws Exception {
 
     List<String> backendZNodeNames = curatorClient.getChildren().forPath(backendAssociationPath);
     for(String backendZNodeName: backendZNodeNames) {
+      logger.debug("Found associations for : " + backendZNodeName);
       String backendZNodePath = getZNodePathForBackend(backendZNodeName);
       List<String> processGroupNodes = curatorClient.getChildren().forPath(backendZNodePath);
       Set<Recorder.ProcessGroup> processGroups = new HashSet<>();
 
       for(String processGroupZNodeName: processGroupNodes) {
         String processGroupZNodePath = getZNodePathForProcessGroup(backendZNodeName, processGroupZNodeName);
         Recorder.ProcessGroup processGroup = Recorder.ProcessGroup.parseFrom(ZookeeperUtil.readZNode(curatorClient, processGroupZNodePath));
+        logger.debug("\t" + processGroup.getAppId() + ", " + processGroup.getCluster() + ", " + processGroup.getProcName());
```

Comment:
Support for log4j2 is added, so let's use bracket notation rather than string concat. For existing log statements we can do a refactoring later